racetrack_client/README.md

```diff
@@ -1,30 +1,93 @@
-# Racetrack client context
-`racetrack-client` is a CLI client tool for deploying workloads to Racetrack.
+# Racetrack CLI client
+`racetrack-client` is a CLI client tool for deploying and managing workloads in Racetrack.
 
 Racetrack system allows to deploy jobs in a one step.
 It transforms your code to in-operation workloads, e.g. Kubernetes workloads.
 You write some code according to a set of coventions, you include the manifest file which explains the code, 
 and you submit it to Racetrack. A short while after, the service calling your code is in operation.
 
-# Installation
+## Quickstart
+```shell
+# Install racetrack-client
+pip3 install racetrack-client
```

Comment:
I have an old habit of using `python3 -m pip install --upgrade racetrack-client`. Some people have problems in their python paths and wonky pip installations, and there might be an old version installed so a forced `--upgrade` can one day be helpful.